nowplaying/options.py

```diff
@@ -0,0 +1,72 @@
+import configargparse
+
+
+class Options(object):
+    """Contain all hardcoded and loaded from configargparse options."""
+
+    """How many seconds the main daemon loop sleeps."""
+    sleepSeconds = 1
+
+    """Default socket of 2 minutes, to prevent endless hangs on HTTP requests."""
+    socketDefaultTimeout = 120
+
+    def __init__(self):
+        """Configure configargparse."""
+        self.__args = configargparse.ArgParser(
+            default_config_files=["/etc/nowplaying/conf.d/*.conf", "~/.nowplayingrc"]
+        )
+        self.__args.add_argument(
+            "-m",
+            "--icecast-base",
+            dest="icecastBase",
+            help="Icecast base URL",
+            default="http://stream-master.audio.int.rabe.ch:8000/admin/",
+        )
+        self.__args.add_argument(
+            "--icecast-password", dest="icecastPassword", help="Icecast Password"
+        )
+        self.__args.add_argument(
+            "-i",
+            "--icecast",
+            action="append",
+            help="Icecast base URL, allowed multiple times",
+            default=[
+                "http://stream-master.audio.int.rabe.ch:8000/admin/metadata.xsl?mount=/livestream/rabe-hd.mp3",
+                "http://stream-master.audio.int.rabe.ch:8000/admin/metadata.xsl?mount=/livestream/rabe-high.mp3",
+                "http://stream-master.audio.int.rabe.ch:8000/admin/metadata.xsl?mount=/livestream/rabe-mid.mp3",
+                "http://stream-master.audio.int.rabe.ch:8000/admin/metadata.xsl?mount=/livestream/rabe-low.mp3",
+                "http://stream-master.audio.int.rabe.ch:8000/admin/metadata.xsl?mount=/livestream/rabe-ultra-low.mp3",
+                "http://stream-master.audio.int.rabe.ch:8000/admin/metadata.xsl?mount=/livestream/rabe-high.opus",
+                "http://stream-master.audio.int.rabe.ch:8000/admin/metadata.xsl?mount=/livestream/rabe-low.opus",
+            ],
+        )
+        self.__args.add_argument(
+            "-d",
+            "--dab",
+            action="append",
+            help="DAB audio companion base URL, allowed multiple times (ie. http://dab.example.org:8080)",
+            default=[],
```

Comment:
This config is currently empty out of the box due to a lack of auth on the backend. I don't think I spammed the interface with any wrong info before I noticed.